include/pybind11/detail/common.h

```diff
@@ -11,7 +11,7 @@
 
 #define PYBIND11_VERSION_MAJOR 2
 #define PYBIND11_VERSION_MINOR 6
-#define PYBIND11_VERSION_PATCH dev0
+#define PYBIND11_VERSION_PATCH 1.dev0
```

Comment:
The combined version needs to be PEP 440 compliant, since it is read in version.py.

Not sure why this ended up `1.` instead of `0.`, though. I think I intended to write `0.dev1`.

Reply:
(fixed)